src/Workspaces/Core/Portable/Shared/Extensions/SemanticModelExtensions.cs

```diff
@@ -45,7 +48,7 @@ public static TSymbol GetEnclosingSymbol<TSymbol>(this SemanticModel semanticMod
                 }
             }
 
-            return default;
+            return default!;
```

Comment:
null, since TSymbol is restricted to class?

Reply:
(odd you needed to ! this -- did you add the constraint and then since it was still not working add the constraint?)

Reply:
I think removing the `!` will also work.